frontend/storybook/src/components/entry/homepage/footer.ts

```diff
@@ -0,0 +1,54 @@
+import "@elements/entry/home/TOSORT/create-leftparagraph";
+import "@elements/entry/home/sections/footer-section";
+import "@elements/core/dividers/stripe-along";
+import "@elements/entry/home/social-networks";
+import "@elements/entry/home/TOSORT/footer-kidsafe"; 
+
+  import {footerjigs} from "~/components/entry/homepage/footer/footer-jigs";
+  import {footerhelp} from "~/components/entry/homepage/footer/footer-help";
+  import {footerproduct} from "~/components/entry/homepage/footer/footer-product";
+  import {footercontactus} from "~/components/entry/homepage/footer/footer-contactus";
+  import {footerwhoweare} from "~/components/entry/homepage/footer/footer-whoweare";
+  import {footerproductsservices} from "~/components/entry/homepage/footer/footer-productsservices";
+
+export default {
+  title: 'Homepage',
+}
+
+
+const STR_lightBlue = "lightBlue";
+const STR_TITLEJi="Jewish Interactive";
+const STR_TITLEAbout="About Us";
+const STR_SIZEMEDIUM="medium";
+const STR_PATHKIDSAFE="logo-kid-safe.jpg";
+const STR_TERMS="Terms & Conditions";
+const STR_PRIVACY="Privacy Policy";
+
+export const footer = () => {
+    return `
+    <footer-section >   
+    <title-section titlecolor="${STR_lightBlue}" title="${STR_TITLEJi}" size="${STR_SIZEMEDIUM}" slot="titleJi"></title-section>
+    <title-section titlecolor="${STR_lightBlue}" title="${STR_TITLEAbout}" size="${STR_SIZEMEDIUM}" slot="titleAbout"></title-section>
```

Comment:
The static content should be in elements, not the component.

Reply:
Fixed